primedev/server/weaponx.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "client/weaponx.h"
+
+#pragma pack(push, 1)
+struct CWeaponX
+{
+	void* vftable;
+	const char gap_8[5128];
+	// this is used by the game as the initial offset
+	// for getting eWeaponVar values. No idea about
+	// the actual datatype. Start of another struct?
+	const char weaponVars[0x1000];
```

Comment:
Does native do any checks to ensure that this is within some bounds? if so, you could probably figure out the exact size

Reply:
could probably get the highest offset and then add 8 bytes to that, should be enough for our use case?